modules/sdk/src/main/scala/org/tessellation/sdk/domain/Daemon.scala

```diff
@@ -1,5 +1,20 @@
 package org.tessellation.sdk.domain
 
+import cats.effect.kernel.{Concurrent, Spawn}
+import cats.syntax.functor._
+
+import fs2._
+
 trait Daemon[F[_]] {
   def start: F[Unit]
 }
+
+object Daemon {
+
+  def spawn[F[_]: Spawn](thunk: F[Unit]): Daemon[F] = new Daemon[F] {
+    def start: F[Unit] = Spawn[F].start(thunk).void
+  }
+
+  def drainStream[F[_]: Concurrent](stream: Stream[F, Unit]): Daemon[F] =
+    spawn[F] { stream.compile.drain }
```

Comment:
Don't you think it's an overkill? We'll use .compile.drain in many places not related to Daemon, as well as many other operators that are repeated over the code.

Reply:
Yep, it could be unnecessary, I'll remove it.